packages/compass-aggregations/src/modules/pipeline-builder/pipeline-parser/pipeline-parser.ts

```diff
@@ -99,22 +99,47 @@ type ValidateResponse = {
   errors: PipelineParserError[];
 };
 
+function getOrphanComments(comments: t.Comment[], visitedComments: Set<t.Comment>): t.Comment[] {
```

Comment:
Nit: this seems to be used just one, is it worth having it in a separate method?